spec/models/section_spec.rb

```diff
@@ -165,14 +165,14 @@ def key_classes_for(hash)
   describe "#eql?" do
     let(:editions) { [draft_edition_v1] }
 
-    it "is considered the same as another section instance if they have the same id" do
+    it "is considered the same as another section instance if they have the same uuid" do
```

Comment:
Now we are using UUIDs the description of this spec reads slightly odd - as they are unique by definition, I'd never expect two instances to have the same UUID.